Makefile

```diff
@@ -27,7 +27,7 @@ coverage:
 
 ## format - Formats the project
 format:
-	dotnet format
+	dotnet dotnet-format --no-restore
```

Comment:
Question: Is it not just `dotnet-format` instead of `dotnet dotnet-format`? Sad if it is... redundant

Reply:
`dotnet` brings up the dotnet CLI utility. From there, we tell it to use the `dotnet-format` tool.
You can also summon `dotnet-format` directly if it was installed globally. Our tools are installed locally, so no.